actioncable/Rakefile

```diff
@@ -3,32 +3,33 @@ require 'pathname'
 require 'sprockets'
 require 'coffee-script'
 require 'action_cable'
+require 'blade'
 
 dir = File.dirname(__FILE__)
 
 task :default => :test
+task :test => [ 'test:ruby', 'test:javascript' ]
 
 task :package => "assets:compile"
 task "package:clean" => "assets:clean"
 
-Rake::TestTask.new do |t|
-  t.libs << "test"
-  t.test_files = Dir.glob("#{dir}/test/**/*_test.rb")
-  t.warning = true
-  t.verbose = true
-  t.ruby_opts = ["--dev"] if defined?(JRUBY_VERSION)
-end
-
 namespace :test do
+  Rake::TestTask.new(:ruby) do |t|
+    t.libs << "test"
+    t.test_files = Dir.glob("#{dir}/test/**/*_test.rb")
+    t.warning = true
+    t.verbose = true
+    t.ruby_opts = ["--dev"] if defined?(JRUBY_VERSION)
+  end
+
   task :isolated do
     Dir.glob("test/**/*_test.rb").all? do |file|
       sh(Gem.ruby, '-w', '-Ilib:test', file)
     end or raise "Failures"
   end
 
   task :javascript do
-    require 'blade'
-    Blade.start(interface: :runner)
+    sh 'bundle exec blade ci'
```

Comment:
Think that'll actually break too since we're depending on the `require 'action_cable'` in this file to compile `action_cable.coffee.erb` with correctly (with the INTERNAL data).